<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBoxdata">
     <property name="title">
      <string>启动</string>
     </property>
     <widget class="QCheckBox" name="loadlastCheckbox">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>161</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>打开上一次的标签页</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="openRemoteCheckbox">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>141</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>打开外部链接</string>
      </property>
     </widget>
     <widget class="QLabel" name="msglabel">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>100</y>
        <width>181</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxfont">
     <property name="title">
      <string>字体</string>
     </property>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>70</y>
        <width>105</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>选择</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>241</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>-19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>-20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
